### ISSUE TITLE
starting SAB does not check anymore that SAB is already running?

### ISSUE THREAD

**Issue by sanderjo** [sabnzbd/sabnzbd#747]
Git-version as of Dec 10:

Starting SAB does not check anymore that SAB is already running? Result: two concurrent SABs running on different ports. See below.

Must have to do with check_for_sabnzbd() and is_sabnzbd_running() ... I will check that.

With the 'older' sabnzbd, a second won't be started. The message is strange (as there is no existing browser session), but the check works and avoids a second instance of SAB running.

```
sander@flappie:~$ sabnzbdplus 
sander@flappie:~$ Created new window in existing browser session.
```

FWIW: the check seems to be that SAB tries to connect to itself, as the already running SAB instance says:

`2016-12-10 11:16:44,598::DEBUG::[interface:522] API-call from ::1 [Python-urllib/2.7] {'mode': u'version'}
`

The new SAB:
```
sander@flappie:~$ ps -ef | grep -i sabnzbd
sander    7213  4564  1 11:03 pts/12   00:00:04 /usr/bin/python -OO ./SABnzbd.py
sander    9821 16329  0 11:04 pts/19   00:00:02 /usr/bin/python -OO ./SABnzbd.py
sander   28581 10898  0 11:10 pts/10   00:00:00 grep --color=auto -i sabnzbd
```

... on different ports:
```
sander@flappie:~$ sudo netstat -apon | grep LISTEN | grep -v LISTENING | grep -i python
tcp        0      0 0.0.0.0:8080            0.0.0.0:*               LISTEN      7213/python      off (0.00/0/0)
tcp        0      0 0.0.0.0:8085            0.0.0.0:*               LISTEN      9821/python      off (0.00/0/0)
tcp        0      0 0.0.0.0:9090            0.0.0.0:*               LISTEN      7213/python      off (0.00/0/0)
tcp        0      0 0.0.0.0:9095            0.0.0.0:*               LISTEN      9821/python      off (0.00/0/0)
tcp6       0      0 ::1:8080                :::*                    LISTEN      7213/python      off (0.00/0/0)
tcp6       0      0 ::1:8085                :::*                    LISTEN      9821/python      off (0.00/0/0)
tcp6       0      0 ::1:9090                :::*                    LISTEN      7213/python      off (0.00/0/0)
tcp6       0      0 ::1:9095                :::*                    LISTEN      9821/python      off (0.00/0/0)
```

**Comment by Safihre** [comment]
This has never worked for `develop` versions, because in `is_sabnzbd_running` it checks for a version number:
```python
url = '%s&mode=version' % (url)
ver = sabnzbd.newsunpack.get_from_url(url, timeout=timeout)
return bool(ver and re.search(r'\d+\.\d+\.', ver))
```

**Comment by sanderjo** [comment]
Ah, I see. So "if on a develop version don't check if SAB is already running" is a design decision? Interesting.

**Comment by Safihre** [comment]
It's by design:
https://github.com/sabnzbd/sabnzbd/commit/f5b6203194f17ec3a65417ea00e17c17bb2dbb41
Not sure what could go wrong though..

**Comment by Safihre** [comment]
Problem solved: 48d2fa3
